DuckDuckGo/Bookmarks/Model/BookmarkOutlineViewDataSource.swift

```diff
@@ -264,6 +264,13 @@ final class BookmarkOutlineViewDataSource: NSObject, BookmarksOutlineViewDataSou
         }
 
         let destination = destinationNode.isRoot ? PseudoFolder.bookmarks : destinationNode.representedObject
+
+        if isSearching {
+            let result = dragDropManager.validateDropWhileInSearchMode(info, to: destination)
```

Comment:
should we just keep the change in scope of the `BookmarkOutlineViewDataSource`? It seems it‘s overcomplicating things and spreading the responsibilities to the class not really meant to deal with search – `BookmarkDragDropManager`.
Maybe let‘s just convert it to `guard !isSearching || destination is BookmarkFolder else { return .none }`, WDYT?

Reply:
I agree 👍🏼 . Addressed